casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -419,7 +439,9 @@ class RuntimeManagerImpl[F[_]: Concurrent: Metrics: Span: Log](
       runtime: Runtime[F],
       startHash: StateHash,
       terms: Seq[ProcessedDeploy],
-      replayDeploy: ProcessedDeploy => F[Option[ReplayFailure]]
+      systemDeploys: Seq[ProcessedSystemDeploy],
+      replayDeploy: ProcessedDeploy => F[Option[ReplayFailure]],
+      replaySystemDeploy: ProcessedSystemDeploy => F[Option[ReplayFailure]]
```

Comment:
I don't think either of the `replayDeploy` or `replaySystemDeploy` parameters are necessary, since we use the `isGenesis` flag in `replayComputeState`. The first existed before this PR, so whether you want to remove it is up to you. `replaySystemDeploy` should probably be removed though.

Reply:
Are you saying the parameters can be removed because `replayComputeState` is only used for genesis?